jkube-kit/doc/src/main/asciidoc/inc/enricher/quarkus-healthcheck/gradle/_smallrye_dependency.adoc

```diff
@@ -0,0 +1,4 @@
+[source,groovy,indent=0,subs="verbatim,quotes,attributes"]
+----
+implementation 'io.quarkus:quarkus-smallrye-health:*'
```

Comment:
The last `*` might be confusing. The docs for the rest of frameworks don't contain this character.

```suggestion
implementation 'io.quarkus:quarkus-smallrye-health'
```